net-p2p/go-ipfs-bin/files/ipfs.init

```diff
@@ -0,0 +1,17 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+IPFS_PATH=${IPFS_PATH:-/var/lib/ipfs/.ipfs}
+
+pidfile="/run/ipfs.pid"
+command="/usr/bin/ipfs"
+command_args="daemon --enable-gc --migrate"
+command_user="ipfs:ipfs"
+start_stop_daemon_args="--wait 1000 -b -e IPFS_PATH=${IPFS_PATH} \
```

Comment:
I'm confused. Do you really need to explicitly make openrc pass this over? Wouldn't it be enough to `export` it in here?

Reply:
What do you mean? When openrc export something it's also visible to to the service without the need of the `-e` option?

Reply:
I haven't test it in detail but AFAIU the mechanics of it, I don't see why it wouldn't be.

Reply:
Just to confirm, have you tested that it works without it? ;-)

Reply:
Yep, I forgot the `export` keyword. At first I didn't deleted the default path and I thought it was working. Sorry :P